test/base/SpecBase.scala

```diff
@@ -66,6 +66,7 @@ trait SpecBase
   val houseConsignmentIndex: Index  = Index(0)
   val itemIndex: Index              = Index(0)
   def emptyUserAnswers: UserAnswers = UserAnswers(arrivalId, mrn, eoriNumber, basicIe043, Json.obj(), Instant.now())
+  val isXI: Boolean                 = true
```

Comment:
I would prefer to see this removed and dedicated unit tests for when this is true and when this is false